src/dispatchers.ts

```diff
@@ -859,22 +861,75 @@ export function followingFirstCursor() {
     return '0';
 }
 
-export async function outboxDispatcher(
-    ctx: RequestContext<ContextData>,
-    handle: string,
-    cursor: string | null,
+export function createOutboxDispatcher(
+    accountService: AccountService,
+    postService: PostService,
+    siteService: SiteService,
 ) {
-    return {
-        items: [],
-        nextCursor: null,
+    return async function outboxDispatcher(
+        ctx: RequestContext<ContextData>,
+        handle: string,
+        cursor: string | null,
+    ) {
+        ctx.data.logger.info('Outbox Dispatcher');
+
+        const pageSize = Number.parseInt(
+            process.env.ACTIVITYPUB_COLLECTION_PAGE_SIZE || '20',
+        );
+
+        const host = ctx.request.headers.get('host')!;
+        const site = await siteService.getSiteByHost(host);
+        if (!site) {
+            throw new Error(`Site not found for host: ${host}`);
+        }
+
+        const siteDefaultAccount =
+            await accountService.getDefaultAccountForSite(site);
```

Comment:
This returns the old AccountType. I want to clean this but here we are using the Fedify context so can't directly get the account from context. Second, [this method](https://github.com/TryGhost/ActivityPub/blob/main/src/account/account.repository.knex.ts#L158) `getBySite` to get AccountEntity using site is also deprecated.

Reply:
Updated this to use `getBySite`